src/formats/prometheus_text_format.erl

```diff
@@ -172,33 +175,36 @@ render_metric(Name, #'Metric'{
     LString = render_labels(Labels),
     Bytes1 = lists:foldl(
         fun(Bucket, Blob) ->
-            <<Blob/binary, (emit_histogram_bucket(Name, LString, Bucket))/binary>>
+            emit_histogram_bucket(Blob, Name, LString, Bucket)
         end,
-        <<>>,
+        Bytes0,
         Buckets
     ),
-    Bytes2 = <<Bytes1/binary, (render_series([Name, "_count"], LString, Count))/binary>>,
-    Bytes3 = <<Bytes2/binary, (render_series([Name, "_sum"], LString, Sum))/binary>>,
+    Bytes2 = render_series(Bytes1, <<Name/binary, "_count">>, LString, Count),
+    Bytes3 = render_series(Bytes2, <<Name/binary, "_sum">>, LString, Sum),
```

Comment:
Also this could probably just return the result without binding the `Bytes3` variable 🤔